build-system/babel-plugins/babel-plugin-transform-jss/test/fixtures/transform-assertions/should-transform/output.mjs

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export const JSS = {
+  button: {
+    fontSize: 12
+  }
+};
+export const CSS = `.button-0-1-1 {
+  font-size: 12px;
+}`;
+export const useStyles = () => ({"button":"button-0-1-1"});
```

Comment:
Can you refresh me on this: did we decide to do a temp const and return the object here to avoid re-allocation on each call?

Reply:
I left it as a TODO in the description of this bug. We should definitely extract to a variable, but I'd prefer to in a followup PR (it doesn't change any interfaces, its just a perf improvement)

Reply:
extracted to a const

Reply:
Sounds great!